### ISSUE TITLE
codecov.yml is not detected

### ISSUE THREAD

**Issue by kmike** [codecov/support#296]
Hey, 

I've created codecov.yml file in https://github.com/TeamHG-Memex/eli5 using a link from https://codecov.io/gh/TeamHG-Memex/eli5/settings/yaml, but this page still shows "No Codecov YAML file found on master yet", and changes are not applied. There were a few commits pushed to the repo after codecov.yml file is created.

**Comment by stevepeak** [comment]
Please see PR above. Thank you 👍

**Comment by kmike** [comment]
Thanks a lot, I wasn't expected a PR 👍  :)
